releasetasks/templates/desktop/funsize_image.yml.tmpl

```diff
@@ -25,6 +25,8 @@
                 dind: true
             maxRunTime: 1800
             image: "taskcluster/image_builder:1.3.0"
+            onExitStatus:
+                retry: [1, 255]
```

Comment:
There's two images that need this tweak in this file. See line 86.

Reply:
Bah, good catch!